joomla/joomla-cms
# [4.3] Fix filename of finder commonwords txt file

Pull Request for Issue #39182  .

### Summary of Changes
When the language code was removed from the filenames of the translation files, this code was forgotten.


### Testing Instructions
Install a language with commonwords file. Check in #__finder_terms_common if the words are there. After applying the patch, install the language again and check if the words are present now.

### Link to documentations
Please select:
